Added recipes for r-bibtex and r-captioner

tfenne opened the pull request:
Added new recipes for R packages bibtex and captioner.  Both were created with `conda skeleton cran` and then cleaned up.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipes/r-bibtex```, ```recipes/r-captioner```) and found some lint.

Here's what I've got...


For **recipes/r-bibtex**:

 * The recipe could do with some maintainers listed in the `extra/recipe-maintainers` section.
 * There are too few lines.  There should be one empty line at the end of the file.

For **recipes/r-captioner**:

 * The recipe could do with some maintainers listed in the `extra/recipe-maintainers` section.
 * The recipe `license` should not include the word "License".
 * There are too few lines.  There should be one empty line at the end of the file.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes/r-bibtex```, ```recipes/r-captioner```) and found it was in an excellent condition.

ocefpaf commented on recipes/r-captioner/meta.yaml:
```diff
@@ -0,0 +1,44 @@
+
+about:
+  home: https://github.com/adletaw/captioner
+  license: Custom
```
User "Other" instead of `Custom`. (We don't really have a standard for this but I think most of the feedstocks are using "Other").

ocefpaf commented on recipes/r-captioner/meta.yaml:
```diff
@@ -0,0 +1,44 @@
+about:
+  home: https://github.com/adletaw/captioner
+  license: Custom
+  license_url: https://github.com/adletaw/captioner/LICENSE
```
Can you download the file and add it to the recipe? See https://conda-forge.org/docs/meta.html#packaging-the-license-manually

tfenne replied:
Thanks for the feedback @ocefpaf.  I've changed the license to `Other` and linked in the license file as described at that link.  Unfortunately the "license" isn't much of a license.  The entire text is:

```
YEAR: 2015
COPYRIGHT HOLDER: Alathea D Letaw
```

but there's not much I can do about that.

ocefpaf replied:
:confounded: That is always a problem. There is no clarity if we can package and ship that or not :unamused:

tfenne replied:
I was worried you might say that.  It _is_ redistributed via CRAN but I can't say if thats good enough.  I just opened an issue with the author to request they select a recognized license, and offered to help with that.  No idea what will come of that though.

ocefpaf replied:
> It _is_ redistributed via CRAN

Is the author the one that put it there? Sorry, but that does not say much.

>  I just opened an issue with the author to request they select a recognized license, and offered to help with that.

Awesome! Thanks!!

> No idea what will come of that though.

Let's wait  for @bgruening here. He probably knows more about the R-ecosystem than me.

ocefpaf commented:
@bgruening you are the R-Expert :wink:
Can you take a look at these recipes?

tfenne commented:
@ocefpaf success!  The nice captioner folks have updated their repo to include the MIT license, clearing that up.  I've just updated the PR here.

@bgruening any chance you could take a look at this in the next couple of days?

tfenne commented:
Thanks @bgruening I've done as you suggested (from my survey of ~10 random r recipes you and johanneskoester seemed by far the most frequent, so I added you two as recipe maintainers, I hope that's ok).  I've also squashed all the commits to tidy things up.
